app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -518,6 +519,16 @@ public void onFsViewerConfig(FilesystemViewerData.Options dopt) {
                 CoolExperimentalStuff.showSpeedReadDialog(activity, _hlEditor.getText().toString());
                 return true;
             }
+            case R.id.action_full_screen:{
+                _isFullScreen = ! _isFullScreen;
+                DocumentActivity temp = (DocumentActivity) activity;
```

Comment:
activity == MainActivity and not DocumentActivity in the quicknote and todo tabs. This will error out if you don't check that here. 

The option should be disabled in main activity.

Reply:
Thanks for pointing out that! Is there are any other faults like this, I will revise it.